docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -30,8 +30,8 @@ set the BIOS settings as follows:
 - **Turbo Mode**: DISABLE
 - **CPU AES**: ENABLE
 
-To test if your settings are correct, you may use the [attestation tool] 
-([binary]) for testing remote attestation against Intel SGX's 
+To test if your settings are correct, you may use the [attestation tool]
```

Comment:
Note that this tool only works for EPID currently.

Reply:
Ok. Moved this tool at the end of the guide (i think it makes morse sense there) and mentioned that it only supports EPID currently.

Would it be straightforward to also add support for DCAP?